src/plugins/query_enhancements/server/search/index.ts

```diff
@@ -4,5 +4,6 @@
  */
 
 export { pplSearchStrategyProvider } from './ppl_search_strategy';
+export { pplRawSearchStrategyProvider } from './ppl_raw_search_strategy';
```

Comment:
nit: I dont see this being referenced anywhere

Reply:
it probably wasn't required but i just moved it to the right location it was previously being exported differently than the other search strategies.,